src/test/munit/headers/deserializing/header-array-default-deserializing-test-suite.xml

```diff
@@ -607,7 +607,7 @@
         </munit:validation>
     </munit:test>
 
-    <munit:test name="success-including-quoted-datetime-array-header-default">
+    <munit:test name="fail-with-quoted-datetime-array-header-default">
```

Comment:
Can you explain why these tests changed from success to failure? Was the previous success a bug?

Reply:
Correct, this is a (breaking) change to keep compatibility with the way the rest of the parameter values are validated. Quotes are not required to be added by the developer anymore, instead, they are considered to be part of the value. That's why the test has to fail (the date between quotes does not have a valid date format).
However, when the CSV deserializing strategy is enabled, quotes are required to delimit values that might have delimiters inside it, so quotes in that case are not considered to be part of the value.
Thanks for asking @iglosiggio, we can continue to discuss this with the rest of the team to agree if this is the behavior that we want, CCs @JuanAller @juanchib